.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -227,6 +280,7 @@ jobs:
           done
 
       - name: Install Cilium ${{ steps.vars.outputs.downgrade_version }} (${{ matrix.name }})
+        if: ${{ steps.vars.outputs.downgrade_version != '' }}
```

Comment:
It seems this line is missing in the `Wait for images to be available` step above. Not intentional I guess?

Reply:
Right, this is a mistake. Resulting from the fact my first version skipped waiting for the images entirely for upgrades from patch versions, as we discussed [earlier](https://github.com/cilium/cilium/pull/28815#discussion_r1401011688). I'll send a fix, thanks!